ui/raidboss/data/07-dt/raid/r6s.ts

```diff
@@ -127,14 +127,17 @@ const triggerSet: TriggerSet<Data> = {
       outputStrings: {
         avoidCleave: {
           en: 'Be on boss hitbox (avoid tank cleaves)',
+          ja: 'ボスの足元へ（タンク攻撃を避ける）',
```

Comment:
My DeepL translator says this is “at the boss's feet”, which should be the boss's hitbox boundary, please check.

![image](https://github.com/user-attachments/assets/9f383ae2-9a2a-4fb9-9d3e-5d7fb44d09e4)